docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: July 3, 2019
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2019
+#####################################
+
+Hi, WTD friends, and welcome to the July newsletter! It's Beth A here writing from sweltering Europe, where I've been learning great new words like *canicule* and *Hitzewelle* - aka heatwave in French and German.
+
+(Fun fact: *canicule* comes from the Latin *dies caniculares*, or "dog days" - the most uncomfortable part of summer, so named because they come after the `helical rising <https://en.wikipedia.org/wiki/Heliacal_rising>`_ of Sirius, the Dog Star. Well, I thought it was interesting.)
```

Comment:
This was a bit of an experiment. Tell me if you hate it and I'll take it out.

Reply:
CAN I SAY I LOVE IT INSTEAD? :HEART_EYES:

Reply:
🙌

Reply:
Sweet!